<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiptViewer</class>
 <widget class="QWidget" name="ReceiptViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="dateGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Dates</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Execution</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Payment</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateExecution">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>118</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="datePayment">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Banked</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateBanked">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Book value</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateBook">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="amoutsGroup">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>286</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Amount</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="amountsView">
      <property name="editTriggers">
       <set>QAbstractItemView::AllEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
